app/views/shared/_head.haml

```diff
@@ -3,7 +3,7 @@
 %meta{content: "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
 %meta{content: "width=device-width, initial-scale=1.0, maximum-scale=1", name: "viewport"}/
 != meta_data_tags
-= canonical_tag(current_store.url)
+%link{ href: current_store.url, rel: 'canonical' }/
```

Comment:
@krzysiek1507 yes I checked it and it crashes on `to_sym` method on `request.params['action']` which are missing on error pages